onnxruntime/core/providers/openvino/openvino_execution_provider.h

```diff
@@ -16,16 +16,23 @@
 
 namespace onnxruntime {
 
+struct OVDevices {
+  ov::Core core;
+  std::vector<std::string> ovDevices() const {
```

Comment:
The naming convention does not seem right

Reply:
it should get_ov_devices()

Reply:
fixed